cpp/modmesh/buffer/pymod/wrap_SimpleArray.cpp

```diff
@@ -149,35 +149,17 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapSimpleArray
                 { return self.reshape(make_shape(shape)); })
             .def(
                 "transpose",
-                [](wrapped_type & self, py::object const & axis, bool const & inplace) -> py::object
+                [](wrapped_type & self, py::args const & axis, bool const & inplace)
                 {
-                    if (inplace)
-                    {
-                        if (axis.is_none())
-                        {
-                            self.transpose();
-                        }
-                        else
-                        {
-                            self.transpose(make_shape(axis));
-                        }
-                        return py::none();
-                    }
-
-                    wrapped_type self_copy(self);
-                    if (axis.is_none())
-                    {
-                        self_copy.transpose();
-                    }
-                    else
-                    {
-                        self_copy.transpose(make_shape(axis));
-                    }
-                    return py::cast(std::move(self_copy));
+                    wrapped_type * ret = inplace ? &self : new wrapped_type(self);
```

Comment:
Ternary expression will make it harder to use a debugger to step.  But it's fine for now.

Reply:
Silence it using `NOLINT` comment directive.

And next time please also include a copy of the error messages, because GitHub Actions do not keep logs forever.  Having the error messages following the code makes it efficient for discussion.